### PR TITLE
resolves #5618

### LINKED ISSUE #5618: [1.7] kube-proxy flags needs to be updated
**This is a...** 

- [ ] Feature Request
- [X] Bug Report

**Problem:**
Reported by https://github.com/kubernetes/kubernetes/pull/48625#issuecomment-331868843, the kube-proxy reference configuration is missing the `--metrics-bind-address` option.

See https://kubernetes.io/docs/admin/kube-proxy/.

**Proposed Solution:**
Regenerate kube-proxy flags on that page. 

**Page to Update:**
https://kubernetes.io/docs/admin/kube-proxy/

cc @alexbrand

### PR DESCRIPTION
**Pull request by heckj**
regenerated kube-proxy documentation from latest 1.7 branch source and
copied across to this repository

 - resolves #5618

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5824)

### CODE REVIEW

File: docs/admin/kube-proxy.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ notitle: true
 
 The Kubernetes network proxy runs on each node. This
 reflects services as defined in the Kubernetes API on each node and can do simple
-TCP, UDP stream forwarding or round robin TCP, UDP forwarding across a set of backends.
```
Comment (xiangpengzhao) [inline docs/admin/kube-proxy.md]: It's good to keep the white space here.

File: docs/admin/kube-proxy.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ notitle: true
...
-Service cluster IPs and ports are currently found through Docker-links-compatible
+TCP, UDP stream forwarding or round robin TCP,UDP forwarding across a set of backends.
+Service cluster ips and ports are currently found through Docker-links-compatible
```
Comment (xiangpengzhao) [inline docs/admin/kube-proxy.md]: Should we use lower case `ip` here?

Comment (heckj) [reply docs/admin/kube-proxy.md]: All of this content is from generated source - like the whitespace request, relevant changes will need to be in kubernetes/kubernetes, but I'm not entirely sure where. I'll research this further and see where changes need to be made.

File: docs/admin/kube-proxy.md (hunk 1 above)
Comment (heckj) [reply docs/admin/kube-proxy.md]: All of this content is from generated source - the PR to add in whitespace here will really need to be in kubernetes/kubernetes, but I'm not entirely sure where. I'll research this further and see where changes need to be made.

Comment (xiangpengzhao) [reply docs/admin/kube-proxy.md]: Thanks @heckj !

Comment (xiangpengzhao) [reply docs/admin/kube-proxy.md]: FYI: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L339-L340

File: docs/admin/kube-proxy.md (hunk 2 above)
Comment (heckj) [reply docs/admin/kube-proxy.md]: @xiangpengzhao https://github.com/kubernetes/kubernetes/pull/53671 has been opened to solve upstream issue here

File: docs/admin/kube-proxy.md (hunk 1 above)
Comment (heckj) [reply docs/admin/kube-proxy.md]: @xiangpengzhao https://github.com/kubernetes/kubernetes/pull/53671 has been opened to solve upstream issue here

**Pull request merged**